docs/content/ZooKeeper.md

```diff
@@ -10,6 +10,51 @@ Druid uses [ZooKeeper](http://zookeeper.apache.org/) (ZK) for management of curr
 4.  [Overlord](Indexing-Service.html) leader election
 5.  [Indexing Service](Indexing-Service.html) task management
 
+
+### Property Configuration
```

Comment:
looks like this section was added back by mistake, I think we recently moved it somewhere else.